src/wallet/rpcwallet.cpp

```diff
@@ -2991,7 +2992,8 @@ UniValue zc_raw_joinsplit(const UniValue& params, bool fHelp)
     mtx.nVersion = 2;
     mtx.joinSplitPubKey = joinSplitPubKey;
 
-    JSDescription jsdesc(*pzcashParams,
+    JSDescription jsdesc(false,
```

Comment:
Why do we choose false here?

Reply:
This API is deprecated, so we aren't bothering to make a breaking change to its API.